README.md

```diff
@@ -25,3 +25,13 @@ yarn workspace server run serve
 # Start client and server
 yarn run dev
 ```
+
+
+## Working with Docker
+```
+# Docker build
+docker build . -t <image_name>
```

Comment:
Why not using `TERArium` instead of `<image_name>`?

Reply:
I am overly paranoid, I don't want people to copy-and-paste and accidentally create a mess via namespace shenanigans.